### ISSUE TITLE
Test-SdnKINetworkInterfacePlacement incorrectly triggers warning

### ISSUE THREAD

**Issue by arudell** [microsoft/SdnDiagnostics#146]
**Describe the bug**
In scenarios where the name within servers API does not match the hyper-V host name, function will throw non-accurate warnings:

```
[NODE] Validate placement of network interfaces between Network Controller and Hypervisor
WARNING: [NODE] 020100000108: Network Controller believes [NetBIOS] exists on [GUID] while hypervisor is reporting it exists on [NetBIOS]
```

**Expected behavior**
Need to gracefully be able to handle if the resourceName does not align to the netBIOS name of the server.

**Screenshots**

**Environment Details (please complete the following information):**
 - Azure Stack HCI: 20348.859.amd64fre.fe_release_svc_prod2.220707-1832
 - SdnDiagnostics Version: N/A

**Additional context**
N/A

**Comment by arudell** [comment]
We have removed this test at this time.
